### ISSUE TITLE
Finaliser et valider les liens NOMENCLATURES<>TAXONOMIE

### ISSUE THREAD

**Issue by camillemonchicourt** [PnX-SI/GeoNature#263]
Puis définir les valeurs par défaut : https://github.com/PnX-SI/GeoNature/issues/249

**Comment by gildeluermoz** [comment]
Ce travail est a conduire avec des thématiciens

**Comment by camillemonchicourt** [comment]
On met de côté ou certains peuvent déjà les compléter ?

**Comment by gildeluermoz** [comment]
Le MNHN semblait intéressé pour y travailler. A t-on des retours ?

**Comment by camillemonchicourt** [comment]
On ne leur a pas transmis pour le moment.
Une relecture/complément de Donovan aurait été intéressante mais sinon ça viendra plus tard.

**Comment by camillemonchicourt** [comment]
OK merci.

**Comment by DonovanMaillard** [comment]
Un socle assez important avait été fait à l'époque, sujet non repris et besoin non remis en avant depuis plus de 7 ans. Rien de planifié sur le sujet dans l'immédiat

**Comment by camillemonchicourt** [comment]
Oui c'est dommage de ne pas l'avoir complété et affiné.
Et je crois qu'on a ajouté des nomenclatures depuis, sans les associer ou très partiellement aux rangs ou groupes pour lesquels ils sont pertinents.

Je trouve dommage que ce travail n'est pas été mené au bout.

**Comment by DonovanMaillard** [comment]
Oui, on avait quand même fait énormément de choses sur le de maximum de groupes. Là ou ce n'est pas fait, c'est sur des groupes qu'aucun de nous ne maitrisait. Dans le principe c'est dommage de ne pas être allés au bout, maintenant passer énormément de temps pour attribuer chaque valeur, de chaque nomenclature, à des groupes sur lesquels personne ne fait réellement de saisies, c'est pas forcément pertinent non plus. Surtout que le seul impact, c'est que toutes les valeurs de nomenclatures sont proposées sans être pré-filtrées, même ça n'empêche pas la saisie donc les groupes sont quand même pris en charge correctement dans l'outil.
